engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/migration/MigrationPlanReportDto.ftl

```diff
@@ -0,0 +1,14 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/migration/validate-migration-plan/index.html -->
+<#macro dto_macro docsUrl="">
+<@lib.dto >
+    
+    <@lib.property
+        name = "instructionReports"
+        type = "array"
```

Comment:
The `MigrationInstructionValidationReportDto` is missing. The file should be created, and added here:
```suggestion
        type = "array"
        dto = "MigrationInstructionValidationReportDto"
```